.github/workflows/post-merge.yaml

```diff
@@ -69,10 +120,11 @@ jobs:
         run: make ci-release-test
         timeout-minutes: 60
 
-      - name: Build Release Binaries
-        env:
-          TELEMETRY_URL: ${{ secrets.TELEMETRY_URL }}
-        run: make release-local
+      - name: Download release binaries
```

Comment:
Just for my own understanding...whereas before we would build the OPA binaries here, now they will be built above by `release-build` and `release-build-darwin` and then downloaded here. Is that correct?

Reply:
Yup!